Canvas/canvas_scraper.py

```
@@ -0,0 +1,139 @@
+        filename = url.split('/')[-1].split('?')[0]
+        path = os.path.join(output_dir, filename)
+        with open(path, 'wb') as f:
+            for chunk in resp.iter_content(chunk_size=8192):
+                if chunk:
+                    f.write(chunk)
+        return path
+
+
+def _demo() -> None:
+    base = "https://learning.unisg.ch"
+    scraper = CanvasScraper(base)
```

Ruff (0.11.9)

Local variable `scraper` is assigned to but never used

Remove assignment to unused variable `scraper`

(F841)